packages/http-logger/package.json

```diff
@@ -1,32 +1,32 @@
-{
-  "name": "@sliit-foss/http-logger",
-  "version": "1.3.1",
-  "description": "Http logging middleware for Express.js",
-  "main": "dist/index.js",
-  "scripts": {
-    "build": "node ../../scripts/esbuild.config.js",
-    "build:watch": "bash ../../scripts/esbuild.watch.sh",
-    "bump-version": "bash ../../scripts/bump-version.sh --name=@sliit-foss/http-logger",
-    "lint": "bash ../../scripts/lint.sh",
-    "release": "bash ../../scripts/release.sh",
-    "test": "bash ../../scripts/test/test.sh"
-  },
-  "dependencies": {
-    "@sliit-foss/module-logger": "1.3.1"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/sliit-foss/npm-catalogue.git"
-  },
-  "homepage": "https://github.com/sliit-foss/npm-catalogue/blob/main/packages/http-logger/readme.md",
-  "keywords": [
-    "logging",
-    "http-logging",
-    "middleware"
-  ],
-  "author": "SLIIT FOSS",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/sliit-foss/npm-catalogue/issues"
-  }
-}
+{
+  "name": "@sliit-foss/http-logger",
+  "version": "1.3.1",
+  "description": "Http logging middleware for Express.js",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "node ../../scripts/esbuild.config.js",
+    "build:watch": "bash ../../scripts/esbuild.watch.sh",
+    "bump-version": "bash ../../scripts/bump-version.sh --name=@sliit-foss/http-logger",
+    "lint": "bash ../../scripts/lint.sh",
+    "release": "bash ../../scripts/release.sh",
+    "test": "bash ../../scripts/test/test.sh"
+  },
+  "dependencies": {
+    "@sliit-foss/module-logger": "workspace:*"
```

Comment:
Let's not go with this approach here, the reason it it becomes difficult to track which packages are updated and which packages aren't if we go with this, since there's a lot of interdependencies between these yes it's easier during development but to the consumer it can become difficult during debugging